dora/core/server/worker/src/main/java/alluxio/worker/http/HttpServerHandler.java

```diff
@@ -178,6 +185,55 @@ private HttpResponseContext doListFiles(HttpRequest httpRequest, HttpRequestUri
     }
   }
 
+  private HttpResponseContext doLoad(HttpRequest httpRequest, HttpRequestUri httpRequestUri) {
+    HttpLoadOptions.Builder builder = HttpLoadOptions.Builder.newBuilder();
+
+    Map<String, String> parameters = httpRequestUri.getParameters();
+    String opTypeStr = parameters.get("opType");
+    if (opTypeStr != null && !opTypeStr.isEmpty()) {
+      builder.setOpType(HttpLoadOptions.OpType.of(opTypeStr));
+    }
+    String partialListingStr = parameters.get("partialListing");
+    if (partialListingStr != null && !partialListingStr.isEmpty()) {
+      builder.setPartialListing(Boolean.parseBoolean(partialListingStr));
+    }
+    String verifyStr = parameters.get("verify");
+    if (verifyStr != null && !verifyStr.isEmpty()) {
+      builder.setVerify(Boolean.parseBoolean(verifyStr));
+    }
+    String bandwidthStr = parameters.get("bandwidth");
+    if (bandwidthStr != null && !bandwidthStr.isEmpty()) {
+      builder.setBandWidth(Long.parseLong(bandwidthStr));
+    }
+    String verboseStr = parameters.get("verbose");
+    if (verboseStr != null && !verboseStr.isEmpty()) {
+      builder.setVerbose(Boolean.parseBoolean(verboseStr));
+    }
+    String loadMetadataOnlyStr = parameters.get("loadMetadataOnly");
+    if (loadMetadataOnlyStr != null && !loadMetadataOnlyStr.isEmpty()) {
+      builder.setLoadMetadataOnly(Boolean.parseBoolean(loadMetadataOnlyStr));
+    }
+    String skipIfExistsStr = parameters.get("skipIfExists");
+    if (skipIfExistsStr != null && !skipIfExistsStr.isEmpty()) {
+      builder.setSkipIfExists(Boolean.parseBoolean(skipIfExistsStr));
+    }
+    String progressFormatStr = parameters.get("progressFormat");
+    if (progressFormatStr != null && !progressFormatStr.isEmpty()) {
+      builder.setProgressFormat(progressFormatStr);
+    }
+    String path = parameters.get("path");
+    path = handleReservedCharacters(path);
+
+    String responseStr = mLoadService.load(new AlluxioURI(path), builder.build());
+
+    FullHttpResponse response = new DefaultFullHttpResponse(httpRequest.protocolVersion(), OK,
+        Unpooled.wrappedBuffer(responseStr.getBytes()));
+    response.headers()
+        .set(CONTENT_TYPE, APPLICATION_JSON)
```

Comment:
The actuall returned content is in plain text instead of JSON, as it appears in your screenshot.

Reply:
Updated, I replaced it with `TEXT_PLAIN`.